src/term/mod.rs

```diff
@@ -147,14 +147,14 @@ impl<'a> RenderableCellsIter<'a> {
             line: self.cursor.line,
             column: self.cursor.col,
             inner: original_cell,
-        });
+        }).unwrap();
 
         // Prints the cursor
         self.cursor_cells.push_back(Indexed {
             line: self.cursor.line,
             column: self.cursor.col,
             inner: cursor_cell,
-        });
+        }).unwrap();
```

Comment:
Under which condition does this fail? Does it make sense to add error-checking here?

Reply:
I imagine like it fails when the underlying array is already full. To me it seems fine to make the claim that we will never push beyond its capacity. `expect("won't exceed capacity")` would be preferred to `unwrap()`, though.

Reply:
Yes, it only fails when full. `unwrap()` has been replaced by `.expect()`.